tools/packaging/common/istio-start.sh

```diff
@@ -99,10 +99,6 @@ if [ "${ISTIO_INBOUND_INTERCEPTION_MODE}" = "TPROXY" ] ; then
   EXEC_USER=root
 fi
 
-if [ -z "${PILOT_ADDRESS:-}" ]; then
-  PILOT_ADDRESS=istiod.${ISTIO_SYSTEM_NAMESPACE}.svc:${ISTIO_PILOT_PORT}
```

Comment:
I think even ifthis is not set, `discoveryAddress` will be present in L122 and cause issues? It would be great if we have a VM+revisions test to prove this works